apps/web/src/components/Feed/SuggestedProfileCard.tsx

```diff
@@ -0,0 +1,241 @@
+import { useMemo } from 'react';
+import { graphql, useFragment } from 'react-relay';
+import styled from 'styled-components';
+
+import { SuggestedProfileCardFollowFragment$key } from '~/generated/SuggestedProfileCardFollowFragment.graphql';
+import { SuggestedProfileCardFragment$key } from '~/generated/SuggestedProfileCardFragment.graphql';
+import { contexts } from '~/shared/analytics/constants';
+import { removeNullValues } from '~/shared/relay/removeNullValues';
+import { useLoggedInUserId } from '~/shared/relay/useLoggedInUserId';
+import colors from '~/shared/theme/colors';
+import unescape from '~/shared/utils/unescape';
+
+import Badge from '../Badge/Badge';
+import breakpoints from '../core/breakpoints';
+import Markdown from '../core/Markdown/Markdown';
+import { HStack, VStack } from '../core/Spacer/Stack';
+import { BaseM } from '../core/Text/Text';
+import FollowButton from '../Follow/FollowButton';
+import { ProfilePicture } from '../ProfilePicture/ProfilePicture';
+
+type Props = {
+  userRef: SuggestedProfileCardFragment$key;
+  queryRef: SuggestedProfileCardFollowFragment$key;
+  showFollowButton?: boolean;
+  onClick?: () => void;
+};
+
+export default function SuggestedProfileCard({
+  userRef,
+  queryRef,
+  onClick,
+  showFollowButton = true,
+}: Props) {
+  const user = useFragment(
+    graphql`
+      fragment SuggestedProfileCardFragment on GalleryUser {
+        id
+        username
+        badges {
+          name
+          imageURL
+          ...BadgeFragment
+        }
+        bio
+        galleries {
+          tokenPreviews {
+            large
+          }
+          hidden
+        }
+        ...ProfilePictureFragment
+        ...FollowButtonUserFragment
+      }
+    `,
+    userRef
+  );
+
+  const query = useFragment(
+    graphql`
+      fragment SuggestedProfileCardFollowFragment on Query {
+        ...FollowButtonQueryFragment
+        ...useLoggedInUserIdFragment
+      }
+    `,
+    queryRef
+  );
+
+  const loggedInUserId = useLoggedInUserId(query);
+  const isOwnProfile = loggedInUserId && loggedInUserId === user?.id;
+
+  if (!user) {
+    throw new Error('No user available to showcase SuggestedProfileCard');
+  }
+
+  const { bio } = user;
+  const unescapedBio = useMemo(() => (bio ? unescape(bio) : ''), [bio]);
+
+  const bioFirstLine = useMemo(() => {
+    if (!unescapedBio) {
+      return '';
+    }
+    return unescapedBio.split('\n')[0] ?? '';
+  }, [unescapedBio]);
```

Comment:
should be able to shortcut the above few lines:

```js
user.bio?.split('\n')[0] ?? ''
```

Reply:
nice! made the change and checked it doesn't break anything

<img width="538" alt="Screenshot 2024-04-08 at 18 33 48" src="https://github.com/gallery-so/gallery/assets/49758803/87eacd4e-b724-4534-b1b7-d586363c21e4">